semantic/src/site/globals/site.overrides

```diff
@@ -123,6 +123,38 @@ p strong {
   margin-left: 10px;
 }
 
+/* Application home */
+#application-area {
+  font-family: @userAreaFont;
+  display: flex;
+  flex-direction: column;
+}
+
+.company-title .h4 {
+  height: 45px;
+  width: 185px;
+  letter-spacing: @letterSpacingRegular;
+  line-height: @lineHeightBig;
+}
+
+/* TODO: Add issue to trim if app name is too big*/
+.back {
+  align-self: flex-start;
+  width: 132px;
+  height: 20px;
+  font-weight: @regularWeight;
+  font-size: @h5;
+  line-height: @fontHeightRegular;
+}
+
+.rectangle {
```

Comment:
I think more specific class names would be useful, so it's clear what it will be/is being used for (I had to do a "find" on this to know) -- either "what it is" (e.g. `.application-content`), or "what it does" (e.g. `.alert`, `.extra-large`).

Reply:
Done: a58df28